vic/vic_run/src/runoff.c

```diff
@@ -48,7 +48,6 @@ runoff(cell_data_struct  *cell,
     size_t                     time_step;
     int                        last_index;
     int                        tmplayer;
-    size_t                     frost_area;
     int                        fidx;
```

Comment:
My current solution to this is to keep fidx as an int so that the two loops that decrement it can evaluate whether fidx >= 0 (for fidx as size_t, decrementing fidx beyond 0 is meaningless and the loop condition of fidx >= 0 will always be true, at least according to the warning statement).  To avoid warnings on comparing objects of different types, I cast options.Nfrost as int in the 6 or so instances where fidx is involved in an expression with options.Nfrost.

The other solution I looked into was making fidx a size_t but modifying the decrementing loops.  One modification involved adding 1 to the loop start/end values (i.e. fidx = options.Nfrost instead of options.Nfrost - 1; and fidx > 0 instead of fidx >= 0) and subtracting 1 from fidx inside the arrays (e.g. evap[][fidx-1]) but looking at it I felt this would make the code harder to understand / less intuitive.

Hopefully casting options.Nfrost as int is acceptable?